test/server/hot_restart_impl_test.cc

```diff
@@ -66,13 +67,15 @@ class HotRestartImplTest : public testing::Test {
     EXPECT_CALL(os_sys_calls_, bind(_, _, _)).Times(4);
 
     // Test we match the correct stat with empty-slots before, after, or both.
-    hot_restart_ = std::make_unique<HotRestartImpl>(0, 0, "@envoy_domain_socket", 0);
+    hot_restart_ = std::make_unique<HotRestartImpl>(0, 0, socket_addr_, 0);
     hot_restart_->drainParentListeners();
 
     // We close both sockets, both ends, totaling 4.
     EXPECT_CALL(os_sys_calls_, close(_)).Times(4);
   }
 
+  std::string socket_addr_ =
+      absl::StrCat(TestEnvironment::unixDomainSocketDirectory(), "/envoy_domain_socket");
```

Comment:
Is this directory unique for every test? instead of a fixed name, maybe use `TestUtility::uniqueFilename()` to make sure there isn't a conflict

Reply:
> Is this directory unique for every test? instead of a fixed name, maybe use `TestUtility::uniqueFilename()` to make sure there isn't a conflict

Of course [it's not](https://sourcegraph.com/github.com/envoyproxy/envoy/-/blob/test/test_runner.cc?L101), why would anyone want a test tmpdir that they can use as a per-test tmpdir, and/or one that also gets cleaned up after the test completes rather than leaving a mess? I should have known not to expect something like that.

Now using uniqueFilename *as well* as a test tmpdir.